test/integration/TestClient.ts

```diff
@@ -276,4 +324,70 @@ export class TestClient {
       });
     });
   }
+
+  private async removeWindowsDB(dbPath): Promise<void> {
+    await new Promise((resolve: Function, reject: Function) => {
```

Comment:
...since you could just return the Promise instead of awaiting it

```suggestion
    return new Promise((resolve: Function, reject: Function) => {
```

Reply:
Aber das macht doch einen semantischen Unterschied oder nicht?

Reply:
@rrrene Nein, wenn man ein Promise zurückgibt, und dann den Funktionsaufruf `removeWindowsDB ` awaitet. Dann ist das so als würde in der Funktion await stehen. 
Das zurückgeben des Promises bietet sich hier einfach an weil ich nach dem Promise nichts anderes mehr ausführe.

Reply:
Was ich meinte ist: Wenn ich das Promise in der Funktion awaite, dann "warte" ich definitiv an der Stelle. Wenn ich das Promise zurückgebe, ermögliche ich dem Aufrufer, wann er "wartet", weil er auch noch was anderes vorher machen könnte:

```typescript
const returnedPromise  = removeWindowsDB("/foo");
doSomething();
console.log("Oh, i will await the promise now");
await returnedPromise;
```

Reply:
Also: Alles gut, war nur ne Verständnisfrage. Macht das ruhig so.